lib/linux_kernel/processor.c

```diff
@@ -35,78 +35,6 @@
 #define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_LIB
 #include "lib/trace.h"
 
-/**
```

Comment:
I suggest to have all the code in the following MACRO test:
```
#ifdef CONFIG_X86_64

...
...

#endif

So, all code will remain the same for X86 platform.

Reply:
Otherwise, if all of them removed, we have to main two different versions of code,
one for x86 and one for arm.

If the code for x86 is changed, we have to manually merge it to arm.